.gitignore

```diff
@@ -89,4 +89,3 @@ serialsent.raw
 status.txt
 tags
 test.ArduCopter/*
-Thumbs.db
```

Comment:
I understand in the another PR that you comment to remove it. My mistakes, i misunderstood your comment on that PR. I will add it again.